chef/cookbooks/crowbar/templates/default/crowbar-evacuate-host.sh.erb

```diff
@@ -47,7 +47,7 @@ fi
 
 # FIXME: here we need checks for potential errors during the evacuation
 
-while nova list --host "$host" --fields host 2>/dev/null | grep "$host"; do
+while nova list --all-tenants --status active --host "$host" --fields host 2>/dev/null | grep "$host"; do
```

Comment:
Probably get all instances and filter with a whitelist for statuses that we know are troublesome? I cant remember the status of an instance migrating, but its probably not `active` so we could be going forward while there is still instances migrating :/